quattor/filesystems.pan

```diff
@@ -39,7 +39,7 @@ type structure_filesystem = {
     "mount"     : boolean
     "mountopts" : string = "defaults" # "Mount options"
     "type"      : string with match (SELF, "^(auto|ext[2-4]|reiserfs|" +
-                    "reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none)$")
+                    "reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none|proc)$")
```

Comment:
Is it time to re-order this into some sort of logical order? e.g. 
- auto
- none
- proc
- swap
- ext[2-4]
- jfs
- ntfs
- reiser4
- reiserfs
- vfat
- xfs
- zfs